NEWS

```diff
@@ -65,6 +65,7 @@ Master (not on release branches yet)
   via --enable-mpi-cxx.
 - Removed embedded VampirTrace.  It is in maintenance mode since 2013.
   Please consider Score-P (score-p.org) as an external replacement.
+- Remove IB XRC support. 
```

Comment:
Nit: should be clear about removing XRC from openib BTL (i.e., they may still support XRC in MXM and/or UCX).